CGPs/0013.md

```diff
@@ -3,7 +3,7 @@
 - Date: 2020-10-16
 - Author(s): @nambrot
 - Status: DRAFT
```

Comment:
I don't remember off the top of my head, do we update proposal status to something like "PROPOSED" during the governance process?